tests/components/meteo_france/test_config_flow.py

```diff
@@ -0,0 +1,102 @@
+"""Tests for the Meteo-France config flow."""
+from unittest.mock import patch
+
+from meteofrance.client import meteofranceError
+import pytest
+
+from homeassistant import data_entry_flow
+from homeassistant.components.meteo_france.const import CONF_CITY, DOMAIN, SENSOR_TYPES
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_MONITORED_CONDITIONS
+
+from tests.common import MockConfigEntry
+
+CITY = "74220"
+CITY_2 = "69004"
+MONITORED_CONDITIONS = ["temperature", "weather"]
+DEFAULT_MONITORED_CONDITIONS = list(SENSOR_TYPES)
+
+
+@pytest.fixture(name="client")
+def mock_controller_client():
+    """Mock a successful client."""
+    with patch("meteofrance.client.meteofranceClient", update=False):
```

Comment:
@MartinHjelmare Should I patch the library or the HA component ?

Seen library here : https://github.com/home-assistant/home-assistant/pull/30992#discussion_r373869882

Seen HA component here : https://github.com/home-assistant/home-assistant/pull/28968#discussion_r350969540

Reply:
We're patching the library in icloud too. 

I'm not talking about where to patch but what we're patching. We should normally patch in the periphery. Usually that means patching the device library.